README.md

```diff
@@ -1,17 +1,42 @@
-# stactools-package
-
-Template repostitory for [stactools](https://github.com/stac-utils/stactools) packages.
-
-## How to use
-
-1. Clone this template repository as your package name, e.g. `landsat`.
-   This name should be short, memorable, and a valid Python package name (i.e. it shouldn't start with a number, etc).
-   It can, however, include a hyphen, in which case the name for Python imports will be the underscored version, e.g. `landsat-8` goes to `stactools.landsat_8`.
-   Your name will be used on PyPI to publish the package in the stactools namespace, e.g. `stactools-landsat`.
-2. Change into the top-level directory of your package and run `scripts/rename`.
-   This will update _most_ of the files in the repository with your new package name.
-   You'll have to manually update `setup.cfg` and `README.md`. 
-3. Update `setup.cfg` with your package name, description, and such.
-4. Rewrite this README to provide information about how to use your package.
-5. Update the LICENSE with your company's information (or whomever holds the copyright).
-6. Run `sphinx-quickstart` in the `docs` directory to create the documentation template.
+# stactools aafc-landuse
+
+## Agriculture and Agri-Food Canada Land Use
+
+### Description
+
+The Land Use (LU) maps cover all areas of Canada south of 600N at a spatial resolution of 30 metres. The LU classes follow the protocol of the Intergovernmental Panel on Climate Change (IPCC) and consist of: Forest, Water, Cropland, Grassland, Settlement and Otherland.
```

Comment:
`The Land Use (LU) maps cover all areas of Canada south of 600N` Should this be 60degN?

Reply:
That was copied from here https://open.canada.ca/data/en/dataset/9e1efe92-e5a3-4f70-b313-68fb1283eadf, but yes...it should be